roles/front_desk.py

```
@@ -393,6 +401,36 @@ async def cache_message(self, chat_id: str, event: AstrMessageEvent):
         # 6. 将消息添加到 Ledger。上下文清理由压缩策略统一控制，不再因离场状态触发。
         self.context.conversation_ledger.add_message(chat_id, new_message)
 
+        # 7. 登记来源（供 WebUI 认群）：群聊取群名，私聊取发送者昵称，均为上游同步字段。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)